.github/workflows/release.workflow.yaml

```diff
@@ -38,6 +38,12 @@ jobs:
           SEMANTIC_RELEASE_SLACK_WEBHOOK: ${{ secrets.SEMANTIC_RELEASE_SLACK_WEBHOOK }}
         run: npx semantic-release
 
+      - name: Rebase 2-dev (keep linear history)
+        run: |
+          git fetch --no-tags origin 2-dev:2-dev
+          git rebase ${GITHUB_REF_NAME} 2-dev
+          git push origin 2-dev
```

Comment:
Will not work since it require a force push (commit IDs are refresh each time you rebase). Note that the CI user will become the author/co-author of each rebased commit

Reply:
Here that just, add the commit `chore(release): x.x.x-beta.y [skip ci]` created by **semantic-release** to the dev branch, as what is done in other repositories

Reply:
there's no force push as it is fast forward in principle